docs/spec_draft.md

```diff
@@ -343,6 +344,120 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_grad
+
+### Semantics
+
+Computes gradients of `operand`, `scale` and offset, using `mean`, `variance`
```

Comment:
The `offset` seems out pf place. Add a reference to where it belongs

offset (refer to [stablehlo.batchnorminference](#...))

Reply:
I will refer to stablehlo.batch_norm_training that would be more accurate.